sequencer/src/block.rs

```diff
@@ -82,6 +82,10 @@ impl BlockPayload for Payload<TxTableEntryWord> {
     fn builder_commitment(&self, _metadata: &Self::Metadata) -> BuilderCommitment {
         unimplemented!("TODO builder_commitment");
     }
+
+    fn get_transactions(&self, _metadata: &Self::Metadata) -> &Vec<Self::Transaction> {
```

Comment:
Can't we implement this now instead of leaving it unimplemented? I think it can be the same as `transaction_commitments` above on line 77 without the `commit()`.

Reply:
I am trying to use:
```
self.enumerate(metadata)
            .map(|(_, tx)| tx)
            .collect::<Vec<Self::Transaction>>()
            .as_ref()
```
It is throwing `can not return ref` error.

Reply:
So shall I keep this `unimplemented!()` for this PR, because the change will come later?

Reply:
If it doesn't break any tests it's probably fine...although it does make me wonder what this function is for if it's not being called

Reply:
I have a solution that will work. This was the reason I requested the metadata be added to the `get_transactions`. I'll take care of it later.